packages/core/src/components/VerticalNavigation/Navigation/Navigation.tsx

```diff
@@ -433,6 +436,10 @@ export interface HvVerticalNavigationTreeProps
    * target - the behavior when opening an url.
    */
   data?: NavigationData[];
+  /**
+   * Object with the custom css styles to be applied to the popup.
+   */
+  popupStyles?: any;
```

Comment:
There is no need for `any`; we know exactly what we allow here and can provide much better typings. This should be `CSSProperties`.

When in doubt, hover a prop/attribute, and It'll tell you what type it should be.

<img width="604" alt="image" src="https://github.com/lumada-design/hv-uikit-react/assets/638946/1d00f64b-4a6f-4983-abed-38c8597d449f">